client/js/libs/handlebars/localedate.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
```

Comment:
Yeah, this is fine, imo, there is a clear difference between the 2, and we like modularity.